.idea/.name

```diff
@@ -0,0 +1 @@
+imed-app2
```

Comment:
I think you could add all these` .idea` files to gitignore.

Reply:
Done 7b19a0b5ebbbd1688941b012dc10cf481ed5f0ae

Reply:
Didn't work. The files are still here:
![image](https://user-images.githubusercontent.com/1125660/186563033-5832a15d-44ed-46ee-b6fd-30b84016533b.png)

Reply:
Maybe you need to do something like that:
https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Since you already committed those files before adding them to gitignore.

Reply:
Thanks, I believe it worked 2f865c3dd672bb74b0a2d12f4099fe49109d169f

Reply:
![image](https://user-images.githubusercontent.com/1125660/186762235-5b733f67-1bcd-4592-a152-c108ed8f99e2.png)